frontend/src/home/Home.tsx

```diff
@@ -275,32 +243,17 @@ class HomeComponent extends React.Component<Props, HomeState> {
   }
 
   componentDidMount() {
-    // If this is true, then a user is currently logged in and we can fetch their plan
-    if (!this.props.activeSchedule) {
-      const token = getAuthToken();
-      findAllPlansForUser(this.props.userId!, token).then(
-        (plans: IPlanData[]) => {
-          // Once multiple plans are supported, this can be changed to the last used plan
-          let plan: IPlanData = plans[0];
-
-          this.props.setPlanIds(plans.map(plan => plan.id));
-          this.props.setDNDSchedule(plan.schedule);
-          this.props.setMajorPlans(
-            findMajorFromName(plan.major, this.props.majors),
-            plan.coop_cycle
-          );
-          this.props.setPlanName(plan.name);
-          this.props.setLinkSharing(plan.link_sharing_enabled);
-
-          this.setState({
-            fetchedPlan: true,
-            planCount: plans.length,
-          });
-        }
-      );
-    } else if (this.props.activeSchedule) {
-      this.setState({ fetchedPlan: true });
-    }
+    const token = getAuthToken();
+    findAllPlansForUser(this.props.userId!, token).then(
+      (plans: IPlanData[]) => {
+        this.props.setUserPlans(plans, this.props.academicYear);
```

Comment:
This also sets the active plan

Reply:
how do we determine which one is the active plan?